shared/templates/audit_rules_usergroup_modification/bash.template

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ol,multi_platform_rhv,multi_platform_sle,multi_platform_ubuntu,multi_platform_debian
```

Comment:
```suggestion
# platform = multi_platform_all
```

Reply:
I guess multi_platform_all will include openembedded and uos20 products too. Is that ok?

Reply:
Yes it is ok. This rule is not used in the only profile present for these products.

Reply:
I will update the commit, then.